src/views/Layout.jsx

```diff
@@ -13,14 +14,15 @@ import './Layout.css';
 export function Layout() {
 	return (
 		<>
-			<div className="Layout">
+			<div className="Layout h-screen md:max-w-md md:mx-auto">
 				<header className="Layout-header">
 					<h1>Smart shopping list</h1>
 				</header>
 				<main className="Layout-main">
 					<Outlet />
 				</main>
-				<nav className="Nav">
+				<NavWithEffect />
+				{/* <nav className="Nav">
```

Comment:
Is this more commented out code that needs to die? Or is does the /* not mean "block comment this entire bit" in this context?